### DISCUSSION TITLE
Searching Records by email

### DISCUSSION THREAD

**Discussion by LoganPikeDev** [pocketbase/pocketbase#1827] [Q&A] [answered]
I am trying to search the database by a given email address to make sure that accounts with the same email are not allowed to be created. I am trying to use the filters to find the email in the user's collection but can't seem to get it to work. I have temporarily disabled the API rules, yet when I search by email, it still says that admin permission is required. Has anyone tried this sort of email verification?

**Comment by ganigeorgiev** [comment, marked as answer]
> I am trying to search the database by a given email address to make sure that accounts with the same email are not allowed to be created.

But why? The email uniqueness is guaranteed even on db table level. You don't have to check it on your own - you'll get an error response when trying to create a new account with existing email.

Regular users are allowed to search for other users by their email if:
- the request satisfies the "List/Search API" rule of your auth collection
- the users have `emailVisibility` set to true (aka. their email is public)

**Reply by AHCCDevelopment** [reply]
Well that solves my issue then, I was not aware that email uniqueness was guaranteed. Thank you
